scanners/mobilefriendly.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from scanners import utils
+import json
+import os
+import requests
+
+##
+# == pageload ==
```

Comment:
There are a few places in here where the `pageload` stuff from `pageload.py` didn't get replaced, like this one.